cmd/object-api-input-checks.go

```diff
@@ -105,16 +105,8 @@ func checkListMultipartArgs(ctx context.Context, bucket, prefix, keyMarker, uplo
 				KeyMarker:      keyMarker,
 			}
 		}
-		id, err := uuid.Parse(uploadIDMarker)
-		if err != nil {
+		if _, err := uuid.Parse(uploadIDMarker); err != nil {
 			logger.LogIf(ctx, err)
-			return err
-		}
-		if id.IsZero() {
```

Comment:
Can we have the zero check? Because, uuid.Parse will return err as `nil` if uuid has all zeros

```
u,err := uuid.Parse("00000000-0000-0000-0000-000000000000")
```

Reply:
But why do we need this check @Praveenrajmani sending 000 is not really incorrect. ?
IMO it is frivolous to add such checks .

Reply:
Also it doesn't return nil https://play.golang.org/p/XOB_g-9ugo7 because it is a perfectly valid UUID

Reply:
I recommended adding the zero check because it was there already. But as zero is a valid uuid, It makes sense that we could remove the check.